evadb/executor/create_function_executor.py

```diff
@@ -163,6 +164,57 @@ def handle_sklearn_function(self):
             self.node.metadata,
         )
 
+    def handle_xgboost_function(self):
+        """Handle xgboost functions
+
+        We use the Flaml AutoML model for training xgboost models.
+        """
+        try_to_import_xgboost()
+
+        assert (
+            len(self.children) == 1
+        ), "Create sklearn function expects 1 child, finds {}.".format(
+            len(self.children)
+        )
+
+        aggregated_batch_list = []
+        child = self.children[0]
+        for batch in child.exec():
+            aggregated_batch_list.append(batch)
+        aggregated_batch = Batch.concat(aggregated_batch_list, copy=False)
+        aggregated_batch.drop_column_alias()
+
+        arg_map = {arg.key: arg.value for arg in self.node.metadata}
+        from flaml import AutoML
+
+        model = AutoML()
+        settings = {
+            "time_budget": 120,
+            "metric": "r2",
+            "estimator_list": ["xgboost"],
+            "task": "regression",
```

Comment:
We shall make those paramters from arg_map. So like `time_budget` can be configured by the user. You can check ludwig to see how we do that. And we need to add available configurations into the documentation.

Reply:
Adding TIME_LIMIT parameter. Not sure on whether we should add this config for r2/rmse metrics on which to train the model? And if we extend the model to classification as well, then we could add the config for regression/classification as well.